pkgs/by-name/ki/kikoplay/package.nix

```
@@ -0,0 +1,92 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  callPackage,
+  libsForQt5,
+  makeWrapper,
+  qt5,
+  mpv,
+  lua5_3_compat,
+}:
+let
+  qhttpengine = callPackage ./qhttpengine.nix { };
+in
+stdenv.mkDerivation rec {
+  pname = "kikoplay";
+  version = "1.0.3";
+  src = fetchFromGitHub {
+    owner = "KikoPlayProject";
+    repo = "KikoPlay";
+    rev = "${version}";
+    hash = "sha256-y+sT0aHDvKFNaJIxnGxDXZXUiCzuXVF7FPc8t//RMW4=";
+  };
+
+  nativeBuildInputs = [
+    libsForQt5.qmake
+    qt5.wrapQtAppsHook
+    makeWrapper
```

```suggestion
    makeWrapper
    qt5.qtwebsockets # required here by qmake
```
Otherwise the build fails if you enable `strictDeps = true`.